### ISSUE TITLE
How to get Score(probability) of each classes 

### ISSUE THREAD

**Issue by nandini123456** [tpvt99/character-recognition-cnn#2]

**Comment by nandini123456** [comment]
I have 36 classes 0-9 and A-Z.I need know the probability value of each classes during testing the trained model

**Comment by tpvt99** [comment]
oh. If you want to know the probability of each values, I suggest you to make another variable in model() function:
``
prob = tf.nn.softmax(Z)
prob = tf.reduce_mean(prob, axis = 0, keepdims = True)
init = tf.global_variables_initializer()

with tf.Session() as sess:
  sess.run(init)
  for epoch in range(epoches):
    for batch in mini_batches:
      # Do the training here
      # This is the place you put the code for prob:
      real_prob = sess.run(prob, feed_dict  = {X:X_batch, Y: Y_batch)
      print(real_prob) # print to see the probability

**Comment by nandini123456** [comment]
Thank you

**Comment by nandini123456** [comment]
I  trying to add the drop out layer  to each convolution layer, but i didn't know how to add that in forward propagation function. can you please help me out.

**Comment by tpvt99** [comment]
Sure. Let me clone this project to my current host and I will try to rewrite this code. Wait me until tomorrow. I sure will have the edited part for you.

**Comment by nandini123456** [comment]
Thank you

**Comment by tpvt99** [comment]
Hey. Do you mean that add drop out in convolutional layer? Dropout should be added in fully connected layers instead of convolutional layer? Please check again and send me some comments.
